pages/common/pio-project.md

```diff
@@ -0,0 +1,20 @@
+# pio project
+
+> Tool to manage PlatformIO projects.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/project/index.html>.
+
+- Initialize a new PlatformIO based project (defaults to current directory if `--project-dir` is omitted):
+
+`pio project init --project-dir {{path/to/project_directory}}`
+
+- Initialize a new PlatformIO based project, specifying a board ID and several project options:
+
+`pio project init --project-dir {{path/to/project_directory}} --board {{board_id}} --project-option="{{option}}={{value}}}" --project-option="{{option}}={{value}}"`
+
+- Print configurations of a specific project:
+
+`pio project config --project-dir {{path/to/project_directory}}`
+
+- Print data of a specific project, intended for IDE extensions/plugins:
```

Comment:
Would this be useful to a user on the command line / terminal? It strikes me that this might be a bit niche to include here if it's intended for plugins or extensions.

Reply:
I really should focus more on simplicity and usefulness, I'll quickly edit the examples